### ISSUE TITLE
Gridmap does not display all children of tile

### ISSUE THREAD

**Issue by furkanoeztuerk** [godotengine/godot#40550]
**Godot version:**
v3.2.2.stable.official

**OS/device including version:**
Windows 10

**Issue description:**
The gridmap does not display all children of a tile. Only the first meshinstance is displayed, the rest is ignored.
Here is how the tiles scene looks:
![pic1](https://user-images.githubusercontent.com/48966965/87985966-c5e5a980-cadc-11ea-9b3b-2abd63f25955.jpg)
There is one parent node, with two meshinstance children. The first child is a sphere, the second child a cube.

Here is how they look when used in a gridmap:
![pic2](https://user-images.githubusercontent.com/48966965/87986024-ddbd2d80-cadc-11ea-8e1e-04e62992f0e5.jpg)
Notice how only the first meshinstance (sphere) is displayed.

**Steps to reproduce:**
1. Create tilemaps
2. Convert tilemap scene to meshlibrary
3. Use meshlibrary in gridmap

**Minimal reproduction project:**
[Gridmap Bug.zip](https://github.com/godotengine/godot/files/4950040/Gridmap.Bug.zip)

**Comment by TheDuriel** [comment]
Each item only contains one Mesh, this is intended. Should be part of the associated tutorial no?

**Comment by KoBeWi** [comment]
Yeah, the tutorial doesn't really clarify it. It just tells you to download the demo and doesn't even show the basic meshlib tree. I'll transfer it to Godot docs, because it's intended behavior.

Also the Parent node is unnecessary. Your meshes should be directly under root.
